json/src/main/scala/spray/json/JsReader.scala

```diff
@@ -59,8 +59,9 @@ object JsReader {
   }
 
   implicit def option[A: JsReader]: JsReader[Option[A]] = {
-    case JsNull => None
-    case a      => Some(JsReader[A].read(a))
+    case JsAbsent => None
+    case JsNull   => None
```

Comment:
minor point, but you can do `case JsAbsent | JsNull`  here and avoid repeating the RHS (which is trivial, so feel free to ignore me).

Reply:
Noob disclaimer applies.